components/dashboard/src/provider-utils.tsx

```diff
@@ -130,21 +130,30 @@ function attachMessageListener({ onSuccess, onError }: WindowMessageHandler) {
 }
 
 interface OpenOIDCStartWindowParams extends WindowMessageHandler {
-    orgSlug: string;
+    orgSlug?: string;
+    configId?: string;
 }
 
 async function openOIDCStartWindow(params: OpenOIDCStartWindowParams) {
-    const { orgSlug } = params;
+    const { orgSlug, configId } = params;
     let search = "message=success";
     const redirectURL = getSafeURLRedirect();
     if (redirectURL) {
         search = `${search}&returnTo=${encodeURIComponent(redirectURL)}`;
     }
     const returnTo = gitpodHostUrl.with({ pathname: "complete-auth", search }).toString();
+    const searchParams = new URLSearchParams({ returnTo });
```

Comment:
... then append the `activate` param here.

Since https://github.com/gitpod-io/gitpod/pull/17365, this would be necessary pass along in order to have the `config.active` being set on positive outcome, which in turn is used to compute the state of the cell. ;-)